### ISSUE TITLE
a11y: Login form input inertness and tabindex

### ISSUE THREAD

**Issue by tesaguri** [snarfed/bridgy-fed#2514]
The login page (<https://fed.brid.gy/login>) initially shows the list of services[^1], and clicking one of them will display an input field. But the input fields are actually only hidden with `opacity: 0` and thus are interactive even while hidden. So if you attempt to tab around the page, the visually hidden input fields will be focused, which is very confusing.

And in this connection, the list items are not actually `button`s, so keyboard users cannot focus nor click them.

[^1]: Well, it’s not actually of the `list` role, but that’s another story.

**Comment by snarfed** [comment]
Great point, thanks for filing! cc'ing @quillmatiq who's looking at a redesign of our web UI right now.
